themes/classic/_dev/css/components/products.scss

```
@@ -413,9 +413,25 @@
 #blockcart-modal {
   color: $gray-darker;
 
+  .modal-dialog {
+    width: calc(100% - 20px);
+    max-width: 850px;
+  }
+
   .modal-header {
     background: $white;
 
+    .modal-title {
+      font-size: 1rem;
+      font-weight: 400;
+      color: #353943;
```

```suggestion
      color: $gray-dark;
```

Please use SCSS variablese everywhere, the closest one, you can also play with `lighten` and `darken` when needed